src/hw/reverse_linked_list/Solution.java

```diff
@@ -0,0 +1,56 @@
+package hw.reverse_linked_list;
+
+/// HW possible solutions.
+///
+/// @link [Reverse Linked List](https://leetcode.com/problems/reverse-linked-list/description/)
+public class Solution {
+    /// Main.
+    ///
+    /// @param args Arguments list.
+    public static void main(String[] args) {
+        ListNode node_1 = new ListNode(5);
```

Comment:
Please configure checkstyle and use proper variable naming: https://google.github.io/styleguide/javaguide.html#s5.1-identifier-names
```suggestion
        ListNode node1 = new ListNode(5);
```